src/libcore/marker.rs

```diff
@@ -100,13 +100,26 @@ pub trait Sized {
 ///
 /// All implementations of `Unsize` are provided automatically by the compiler.
 ///
+/// `Unsize` is implemented for:
+///
+/// - `[T; N]` is `Unsize<[T]>`
+/// - `T` is `Trait` when `T: Trait`
```

Comment:
Should be `T` is `Unsize<Trait>` when `T: Trait`, in order to be consistent with the above sentence.